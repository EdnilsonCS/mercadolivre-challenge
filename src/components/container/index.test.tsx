import { composeStories } from '@storybook/testing-react';
import { render } from '@testing-library/react';
import * as stories from './index.stories';

const {
  ContainerDefault,
} = composeStories(stories);

describe('Test for container component', () => {
  it('should match snapshot in default style', () => {
    const { container } = render(<ContainerDefault />);

    expect(container).toMatchSnapshot();
  });
});
