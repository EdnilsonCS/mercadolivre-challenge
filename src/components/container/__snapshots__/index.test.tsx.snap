// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test for container component should match snapshot in default style 1`] = `
<div>
  <div>
    <div
      style="background: red;"
    >
      test
    </div>
  </div>
</div>
`;
